recipe/meta.yaml

```diff
@@ -1,25 +1,25 @@
-{%set name = "fbprophet" %}
-{%set version = "0.1.1" %}
-{%set compress_type = "tar.gz" %}
-{%set hash_type = "sha256" %}
-{%set hash_val = "5be931cf0f2c73eb221b269241bb15be0860497511026fd34a7e4da76395f20e" %}
-{%set build_num = "0" %}
+{% set name = "fbprophet" %}
+{% set version = "0.2" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash = "da247d103e5c5aec8ea46ee5a3a56855c9e7c87ba744511e40ec859ac914d463" %}
+{% set build = 0 %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.{{ compress_type }}
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
-  {{ hash_type }}: {{ hash_val }}
+  fn: {{ name }}-{{ version }}.{{ bundle }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
+  {{ hash_type }}: {{ hash }}
 
 build:
-  skip: True  # [win and py27]
-  number: {{ build_num }}
+  number: {{ build }}
   script: python setup.py install --single-version-externally-managed --record=record.txt
   features:
-    - vc14  # [win and py35]
+    - vc 9  # [win and py27]
+    - vc10  # [win and py35]
```

Comment:
Is this correct? I understood vc14 is used for py35 and py36 and vc10 was used for py34.

Reply:
I'll buy that; my change was based on [conda-forge's docs](https://conda-forge.org/docs/buildwin.html#vc-features). Do you have a link? Be nice to put in the recipe...

Reply:
From the docs:
```
build:
  features:
    - vc9   # [win and py27]
    - vc10  # [win and py34]
    - vc14  # [win and py>=35]
```